XCoordinator-Example/Coordinators/AppCoordinator.swift

```diff
@@ -36,20 +32,48 @@ class AppCoordinator: NavigationCoordinator<AppRoute> {
             let viewModel = LoginViewModelImpl(router: unownedRouter)
             viewController.bind(to: viewModel)
             return .push(viewController)
-        case .home:
-            let presentables: [() -> Presentable] = [
-                { HomeTabCoordinator() },
-                { HomeSplitCoordinator() },
-                { HomePageCoordinator() }
-            ]
-            let presentable = presentables[homeRouteTriggerCount % presentables.count]()
-            homeRouteTriggerCount = (homeRouteTriggerCount + 1) % presentables.count
-            return .presentFullScreen(presentable, animation: .fade)
+        case let .home(router):
+            if let router = router {
+                return .presentFullScreen(router, animation: .fade)
+            }
+            let alert = UIAlertController(
+                title: "How would you like to login?",
+                message: "Please choose the type of coordinator used for the `Home` scene.",
+                preferredStyle: .alert)
```

Comment:
I would personally go with actionSheet

Reply:
I will leave it as it is - since it is more prevalent for the user (otherwise it might not be seen directly)